### PR TITLE
APM > .NET > Doc updates

### PR DESCRIPTION
**Pull request by lucaspimentel**
Follow up from #10302

### What does this PR do?
- Reorganize sections what were nested incorrectly. For example, "Configure the Datadog Agent for APM" appeared under "Automatic instrumentation," but it applies to all instrumentation (automatic and custom).
- Add section "Follow the in-app documentation (recommended)" near the top to encourage users to use the in-app docs

### Motivation
We noticed inconsistencies in the documentation which caused users to miss important configuration steps.

### Preview

https://docs-staging.datadoghq.com/lpimentel/apm-dotnet-core/tracing/setup_overview/setup/dotnet-core

### Additional Notes

None

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by jaycdave88 — APPROVED** [review]
lgtm - thanks!

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 1]
```diff
@@ -109,62 +108,63 @@ For a full list of supported libraries, visit the [Compatibility Requirements][1
       curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>.tar.gz \
       | sudo tar xzf - -C /opt/datadog
       ```
-3. Add the [required environment variables](#required-environment-variables).
 
-4. Run the `/opt/datadog/createLogPath.sh` script, which creates a directory for the log files and sets appropriate directory permissions. The default directory for log files is `/var/log/datadog/dotnet`.
+2. Run the `/opt/datadog/createLogPath.sh` script, which creates a directory for the log files and sets appropriate directory permissions. The default directory for log files is `/var/log/datadog/dotnet`.
+
+3. Enable instrumentation in your service by setting the required environment variables.
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:114]:
```suggestion
3. Enable instrumentation in your service by setting the required environment variables. See the section *Instrumenting your service*, below.
```

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 2]
```diff
@@ -40,53 +40,52 @@ further_reading:
...
-3. Run the .NET Tracer MSI installer with administrator privileges.
+1. Download the [.NET Tracer MSI installer][1]. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
 
-4. Stop, then start IIS by running the following commands as an administrator:
+2. Run the .NET Tracer MSI installer with administrator privileges.
 
-    <div class="alert alert-warning"> 
-      <strong>Note:</strong> You must use a stop and start command. This is not the same as a reset or restart command.
-    </div>
+3. Enable instrumentation in your service by setting the required environment variables.
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:75]:
```suggestion
3. Enable instrumentation in your service by setting the required environment variables. See the section *Instrumenting your service*, below.
```

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 3]
```diff
@@ -478,3 +524,5 @@ The following table lists configuration variables for features that are availabl
 [3]: /tracing/setup_overview/custom_instrumentation/dotnet/
 [4]: /getting_started/tagging/unified_service_tagging/
 [5]: /tracing/setup_overview/compatibility_requirements/dotnet-core#integrations
+[6]: https://app.datadoghq.com/apm/docs
+[7]: https://docs.datadoghq.com/agent/basic_agent_usage/
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:528]:
```suggestion
[7]: /agent/basic_agent_usage/
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
GREAT pull request! just a couple minor suggestions

**Review by kayayarai — APPROVED** [review]

**Pull request merged**